draft-ietf-openpgp-pqc.md

```diff
@@ -870,7 +868,7 @@ The algorithm-specific part of the secret key consists of:
 ## Symmetric Algorithms for SEIPD Packets
 
 Implementations MUST implement `AES-256`.
-An implementation SHOULD use `AES-256` in the case of a v1 SEIPD packet, or `AES-256` with any available AEAD mode in the case of a v2 SEIPD packet, if all recipients indicate support for it (explicitly or implicitly).
+An implementation SHOULD use `AES-256` in the case of a v1 SEIPD packet, or `AES-256` with any available AEAD mode in the case of a v2 SEIPD packet, if all communication parties indicate support for it (explicitly or implicitly).
```

Comment:
I think the feedback in the Interim was not to avoid the word "recipient" everywhere. I think here it is sufficiently clear what is meant by "recipient" and I think it's the best word to use. The ambiguity comes in when matching multiple keys/certs to a recipient / entity and it becomes a more abstract concept.

For a PKESK packet there is a recipient / recipient key and for a SEIPD message the recipients are each recipient of the list of PKESK packets. I think that's common wording also in the Crypto Refresh.

Reply:
Yes, it is correct the the term "recipient" is used throughout the crypto-refresh. So we should rather stick to it and probably just avoid conveying the idea of differentiating between keys that belong to a single recipient and to multiple recipients.

Reply:
```suggestion
An implementation SHOULD use `AES-256` in the case of a v1 SEIPD packet, or `AES-256` with any available AEAD mode in the case of a v2 SEIPD packet, if all recipients indicate support for it (explicitly or implicitly).
```